framework/SwiftPlot/Series.swift

```diff
@@ -1,28 +1,38 @@
 // struct defining a data series
-public struct Series {
+public struct Series<T,U> {
     public var barGraphSeriesOptions = BarGraphSeriesOptions()
     public var scatterPlotSeriesOptions = ScatterPlotSeriesOptions()
-    public var points = [Point]()
-    public var scaledPoints = [Point]()
+    public var pairs = [Pair<T,U>]()
```

Comment:
While Pair is a good generic name for a type, I'm not sure that `pairs` as a variable name here is as descriptive as it could be. Maybe `values` would be better.